src/main/java/com/shaft/driver/internal/DriverFactory/OptionsManager.java

```diff
@@ -293,7 +297,14 @@ private ChromiumOptions<?> setupChromiumOptions(ChromiumOptions<?> options, Muta
             options.addArguments("--window-position=0,0", "--window-size=" + DriverFactoryHelper.getTARGET_WINDOW_SIZE().getWidth() + "," + DriverFactoryHelper.getTARGET_WINDOW_SIZE().getHeight());
         }
         if (!SHAFT.Properties.flags.autoCloseDriverInstance()) options.setExperimentalOption("detach", true);
-
+        //Incognito mode for Chrome and Edge
+        if(SHAFT.Properties.web.incognitoMode()) {
+            if(options.getBrowserName().equalsIgnoreCase("chrome")) {
```

Comment:
kindly use the browser name enums instead of using string values for browser names.

Reply:
this change should be implemented inside the "chromiumOptions" method to avoid increasing complexity. kindly move it there.

Reply:
The code is added in the setupChromiumOptions method for both Chrome and Edge browsers
As for firefox it's not a chromium-based browser so the logic was added directly in the setDriverOptions method

As for the Enums approach it's fixed now, can you please check? 

Thanks.